cypress/integration/plugins/alerting-dashboards-plugin/composite_level_monitor_spec.js

```diff
@@ -66,7 +66,7 @@ describe('CompositeLevelMonitor', () => {
       });
     });
 
-    it('by visual editor', () => {
+    xit('by visual editor', () => {
```

Comment:
does x work? shouldn't be it.skip?

can also do something similar and conditional run these tests if its only flaky on `with-security`. Then can still verify on `without-security`

regardless, if you can make a follow up issue that would be great. thank you

Reply:
Yes x also works on excluding these tests. Given that these tests are passing locally, and we are nearing the 2.10 deadline, I'm okay with excluding them from the CI pipeline run